Upcoming-iCal-Events.widget/main.coffee

```diff
@@ -75,17 +99,24 @@ update: (output, domEl) ->
     for i in [0...lines.length-1]
         # Print today subheading
         header = ""
+        day = ''
         if (lines[i+1].startsWith("    today"))
             if (! dom.text().includes('Today'))
                 header = 'Today'
         # Print tomorrow subheading
         else if ( lines[i+1].startsWith("    tomorrow") )
             if (! dom.text().includes('Tomorrow'))
                 header = 'Tomorrow'
-        # Print later subheading
+        # Print day after tomorrow subheading
         else if ( lines[i+1].startsWith("    day after"))
             if (!dom.text().includes('Day After Tomorrow'))
                 header = 'Day After Tomorrow'
+        # Print later subheading
+        else if ( lines[i+1].search(dateRegex) != -1 )
+            day = lines[i+1].trim().substr(0,11)
+            dayHeader = if USE_LATER_DATE then lines[i+1].trim().substr(0,11) else 'Later'
```

Comment:
could re-use the `day` variable here vs repeating the .trim() and .substr()

Reply:
Done.